packages/docsearch-css/src/modal.css

```diff
@@ -576,10 +585,9 @@ assistive tech users */
   }
 
   .DocSearch-Container {
-    height: 100vh;
-    height: -webkit-fill-available;
+    height: max-content;
```

Comment:
I would remove this... but wanted to keep changes minimal. `-webkit-fill-available` should not be used when there is `dvh` that works as expected and is widely supported. `max-content` seems to be the successor...

Reply:
Does any browser support max-content but not calc()? I doubt it, so I would remove this line as it will always be overridden by the next one.

Reply:
Yes, I think that is also good point. Thank you. Fixed!